packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -66,30 +70,40 @@ const classes = computed(() => {
     created: {
       icon: DocumentIconPlus,
       iconClasses: 'icon-dark-jade-400 icon-light-jade-50',
-      showTooltip: false,
     },
     modified: {
       icon: DocumentIconPlusMinus,
       iconClasses: 'icon-dark-orange-400 icon-light-orange-50',
-      showTooltip: false,
     },
     unmodified: {
       icon: CommitIcon,
       iconClasses: 'icon-light-gray-500',
-      showTooltip: true,
+    },
+    noGitInfo: {
+      icon: null,
+      iconClasses: 'icon-light-gray-500',
     },
   }[props.gql?.statusType || 'unmodified']
 })
 
-const gitTooltipSubtext = computed(() => {
+const tooltipMainText = computed(() => {
+  switch (props.gql?.statusType) {
+    case 'unmodified': return props.gql?.subject
+    case 'created': return t('file.git.created')
+    case 'modified': return t('file.git.modified')
+    default: return null
+  }
+})
+
+const tooltipSubtext = computed(() => {
   if (props.gql?.statusType === 'unmodified') {
     return t('specPage.rows.gitTooltipSubtext', {
       author: props.gql.author,
       shortHash: props.gql.shortHash,
     })
   }
 
-  return ''
+  return null
```

Comment:
How come we changed the return type from string to string or null?

Reply:
I guess this was me missing on how JS evaluates falsy strings 😆 . I'm reverting it.